### DISCUSSION TITLE
MQTT HELP!!

### DISCUSSION THREAD

**Discussion by Kareemic** [jomjol/AI-on-the-edge-device#2387] [General] [answered]
So i cant seem to set up connection between mqtt broker and esp. for now i just want to send raw value to my node red mqtt reciever but in digitizer i get this message:
<ERR>	[MQTT IF] Init failed, no handle created! 
i get this error every time. i downloaded mosquitto mqtt and set it up. when i use mqtt spy i can connect to my node-red and send data to my database using said mqtt spy. Im not sure if my mqtt settings in digitizer are right.
![Capture](https://user-images.githubusercontent.com/129008309/235769039-d65f400e-facc-4f6d-9095-ba03a716dca9.JPG)
i really new to this so pls be patient with me. only difference when i put address in mqtt spy i have to remove mqtt:// part. pls can someone explain what i have to do or send a link to a video where it explains a bit. thank you in advance

**Comment by caco3** [comment]
you seem to use a very old verion of our project. Please use the latest one!

**Reply by caco3** [reply]
and the IP you use is impossible!

**Reply by caco3** [reply, marked as answer]
https://www.gutefrage.net/frage/was-heisst-127-0-0-1
